database/src/main/resources/db/migration/V1.50.0__NOTIF-603_add_org_id.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE email_aggregation
+  ADD COLUMN org_id text;
+
+CREATE INDEX email_aggregation_idx ON public.email_aggregation (org_id);
```

Comment:
The index name doesn't seem consistent with the almost consistent names we have so far:
`[theute@fedora database]$ fgrep "CREATE INDEX" * -r
src/main/resources/db/migration/V1.0.0__notifications.sql:CREATE INDEX "IX_account_endpoint_search" ON public.notification_history USING btree (account_id, endpoint_id);
src/main/resources/db/migration/V1.0.0__notifications.sql:CREATE INDEX "IX_time_search_account_mails" ON public.email_aggregation USING btree (account_id, created);
src/main/resources/db/migration/V1.0.0__notifications.sql:CREATE INDEX ix_endpoint_webhooks_endpoint_id ON public.endpoint_webhooks USING btree (endpoint_id);
src/main/resources/db/migration/V1.0.0__notifications.sql:CREATE INDEX ix_endpoints_account_id ON public.endpoints USING btree (account_id);
src/main/resources/db/migration/V1.1.0__applications.sql:CREATE INDEX "IX_application_name"
src/main/resources/db/migration/V1.1.0__applications.sql:CREATE INDEX "IX_event_type_name"
src/main/resources/db/migration/V1.1.0__applications.sql:CREATE INDEX "IX_target_endpoint_id"
src/main/resources/db/migration/V1.33.0__missing_account_id_index.sql:CREATE INDEX ix_event_account_id ON event USING btree (account_id);
src/main/resources/db/migration/V1.33.0__missing_account_id_index.sql:CREATE INDEX ix_behavior_group_account_id ON behavior_group USING btree (account_id);
src/main/resources/db/migration/V1.34.0__NOTIF-410_events_response_time.sql:CREATE INDEX ix_notification_history_event_id ON notification_history (event_id);
src/main/resources/db/migration/V1.37.0__add_subtype_to_endpoint.sql:CREATE INDEX ix_endpoints_type_sub_type ON endpoints (endpoint_type, endpoint_sub_type);
src/main/resources/db/migration/V1.38.0__notification_history_index.sql:CREATE INDEX ix_notification_history_endpoint_id_created
src/main/resources/db/migration/V1.39.0__events_order_by_indexes.sql:CREATE INDEX ix_event_account_id_created_asc
src/main/resources/db/migration/V1.39.0__events_order_by_indexes.sql:CREATE INDEX ix_event_account_id_created_desc
src/main/resources/db/migration/V1.40.0__events_order_by_indexes.sql:CREATE INDEX ix_event_service_index_only_scan_asc
src/main/resources/db/migration/V1.40.0__events_order_by_indexes.sql:CREATE INDEX ix_event_service_index_only_scan_desc
src/main/resources/db/migration/V1.43.0__NOTIF-491_denormalize_events.sql:CREATE INDEX ix_event_index_only_scan ON event (account_id, bundle_id, application_id, event_type_display_name, created DESC, id);
src/main/resources/db/migration/V1.44.0__NOTIF-491_fine_tune_events_optimizations.sql:CREATE INDEX ix_event_account_id ON event (account_id, created DESC, id);
src/main/resources/db/migration/V1.44.0__NOTIF-491_fine_tune_events_optimizations.sql:CREATE INDEX ix_event_account_id_application_id ON event (account_id, application_id, created DESC, id);
target/classes/db/migration/V1.0.0__notifications.sql:CREATE INDEX "IX_account_endpoint_search" ON public.notification_history USING btree (account_id, endpoint_id);
target/classes/db/migration/V1.0.0__notifications.sql:CREATE INDEX "IX_time_search_account_mails" ON public.email_aggregation USING btree (account_id, created);
target/classes/db/migration/V1.0.0__notifications.sql:CREATE INDEX ix_endpoint_webhooks_endpoint_id ON public.endpoint_webhooks USING btree (endpoint_id);
target/classes/db/migration/V1.0.0__notifications.sql:CREATE INDEX ix_endpoints_account_id ON public.endpoints USING btree (account_id);
target/classes/db/migration/V1.1.0__applications.sql:CREATE INDEX "IX_application_name"
target/classes/db/migration/V1.1.0__applications.sql:CREATE INDEX "IX_event_type_name"
target/classes/db/migration/V1.1.0__applications.sql:CREATE INDEX "IX_target_endpoint_id"
target/classes/db/migration/V1.33.0__missing_account_id_index.sql:CREATE INDEX ix_event_account_id ON event USING btree (account_id);
target/classes/db/migration/V1.33.0__missing_account_id_index.sql:CREATE INDEX ix_behavior_group_account_id ON behavior_group USING btree (account_id);
target/classes/db/migration/V1.34.0__NOTIF-410_events_response_time.sql:CREATE INDEX ix_notification_history_event_id ON notification_history (event_id);
target/classes/db/migration/V1.37.0__add_subtype_to_endpoint.sql:CREATE INDEX ix_endpoints_type_sub_type ON endpoints (endpoint_type, endpoint_sub_type);
target/classes/db/migration/V1.38.0__notification_history_index.sql:CREATE INDEX ix_notification_history_endpoint_id_created
target/classes/db/migration/V1.39.0__events_order_by_indexes.sql:CREATE INDEX ix_event_account_id_created_asc
target/classes/db/migration/V1.39.0__events_order_by_indexes.sql:CREATE INDEX ix_event_account_id_created_desc
target/classes/db/migration/V1.40.0__events_order_by_indexes.sql:CREATE INDEX ix_event_service_index_only_scan_asc
target/classes/db/migration/V1.40.0__events_order_by_indexes.sql:CREATE INDEX ix_event_service_index_only_scan_desc
target/classes/db/migration/V1.43.0__NOTIF-491_denormalize_events.sql:CREATE INDEX ix_event_index_only_scan ON event (account_id, bundle_id, application_id, event_type_display_name, created DESC, id);
target/classes/db/migration/V1.44.0__NOTIF-491_fine_tune_events_optimizations.sql:CREATE INDEX ix_event_account_id ON event (account_id, created DESC, id);
target/classes/db/migration/V1.44.0__NOTIF-491_fine_tune_events_optimizations.sql:CREATE INDEX ix_event_account_id_application_id ON event (account_id, application_id, created DESC, id);
`

if I follow the pattern I guess the index should be named `ix_email_aggregation_org_id`

Reply:
indeed. in the last update script we added a new naming pattern I used here as well.